### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by mihaic01**
Add Blast to the list of Ethereum APIs.

### CODE REVIEW

File: src/content/developers/docs/apis/backend/index.md [hunk 1]
```diff
@@ -117,6 +123,7 @@ These libraries abstract away much of the complexity of interacting directly wit
 - [Documentation](https://docs.zmok.io/)
 - [Discord](https://discord.gg/fAHeh3ka6s)
 
+
```
Comment (minimalsm) [inline src/content/developers/docs/apis/backend/index.md:126]:
```suggestion
```

**Review by corwintines — APPROVED** [review]
Thanks @mihaic01!

**Comment by corwintines** [comment]
@all-contributors can you add @mihaic01 for content

**Comment by corwintines** [comment]
Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. We are in the process of getting our 2023 POAPs ready, but once they are you can [claim yours](https://ethereum.org/en/contributing/#poap) also 🥳

**Pull request merged**